generator/api/template/src/test/test.js

```diff
@@ -3,40 +3,16 @@ import 'babel-polyfill';
 import assert from 'assert';
 import GitHubUser from '../lib/GitHubUser';
 
-// WARNING: don't use arrow functions with mocha describe(), it(),
```

Comment:
Let's forget it for now, and all I can find are a few mocha issues where people complained about `this` not working correctly. They're not exactly helpful though for those who might not really understand how `this` context works without arrow functions.